roles/ceph-iscsi-gw/tasks/common.yml

```diff
@@ -2,7 +2,16 @@
 - name: make sure gateway_ip_list is configured
   fail:
     msg: "you must set a list of IPs (comma separated) for gateway_ip_list"
-  when: gateway_ip_list == "0.0.0.0"
+  when:
+    - gateway_ip_list == "0.0.0.0"
+    - not use_new_ceph_iscsi
+
+- name: make sure gateway_iqn is configured
+  fail:
+    msg: "you must set a iqn for the iSCSI target"
+  when:
+    - gateway_iqn | length == 0
```

Comment:
`gateway_iqn` has to be set in `tests/functional/all_daemons/group_vars/iscsigws` and `tests/functional/all_daemons/container/group_vars/iscsigws`